.github/workflows/CI.yml

```
@@ -66,7 +66,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@v1
         with:
-          version: '1.4'
+          version: '1.6'
```

A fair point. At first glance I thought that the doctest errors were unrelated to this PR. FTFY :smile: 

```suggestion
          version: '1.4'
```

FWIW, this PR shouldn't be introducing any doctest changes on its own? Changing the Julia version used for doctests seems like an unrelated change.

---

Fair point